packages/node/src/configure/NodeConfig.ts

```diff
@@ -42,6 +46,14 @@ const DEFAULT_CONFIG = {
   indexCountLimit: 10,
   timestampField: true,
   proofOfIndex: false,
+  networkRegistry: {
+    /* TODO provide a default list */
```

Comment:
This is probably the largest change on the hosted service side. If the project is using the new manifest spec then it only provides a genesis hash, so there needs to be a mapping to the endpoints.